README.md

```diff
@@ -101,12 +92,12 @@ Utilize the following steps to deploy the project
     ansible-galaxy install -r requirements.yml --roles-path=galaxy
     ``` 
  
-3. Update credentials in the _applier/params/community-ninja-board-deployment_ file
+3. Ensure the variables have been updated in the _ninja-board-deployment_ params file
 
 4. Execute the _openshift-applier_
 
     ```
-    ansible-playbook -i applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
+    ansible-playbook -i .applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e="@.openshift/params/ninja-board-deployment
```

Comment:
need double quotes at the end of this line.

Reply:
done